product_tier_validation/models/product_template.py

```diff
@@ -28,9 +28,3 @@ def write(self, vals):
         if "state" in vals:
             vals["active"] = self._is_tier_validated_active(vals["state"])
         return super().write(vals)
-
-    @api.model
-    def _get_default_product_state_id(self):
-        return self.env.ref(
-            "product_state.product_state_draft", raise_if_not_found=False
```

Comment:
where is the draft state then? This module should add it as default, no?

Reply:
Thanks for spotting that. I am updating the `data/product_state_data.xml`.